.ci/e2eKibana.groovy

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent none
+  environment {
+    REPO = 'beats'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    GITHUB_CHECK_E2E_TESTS_NAME = 'E2E Tests'
+    PIPELINE_LOG_LEVEL = "INFO"
+  }
+  options {
+    timeout(time: 3, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    disableConcurrentBuilds()
+  }
+  // http://JENKINS_URL/generic-webhook-trigger/invoke
+  // Pull requests events: https://docs.github.com/en/developers/webhooks-and-events/github-event-types#pullrequestevent
+  triggers {
+    GenericTrigger(
+      genericVariables: [
+        [key: 'action', value: '$.action'],
+        [key: 'pr_id', value: '$.number'],
+        [key: 'pr_base_ref', value: '$.pull_request.base.ref'],
+        [key: 'pr_head_ref', value: '$.pull_request.head.ref'],
+        [key: 'pr_head_sha', value: '$.pull_request.head.sha'],
+        [key: 'pr_title', value: '$.pull_request.title'],
+        [key: 'repo_name', value: '$.repository.name'],
+      ],
+      genericHeaderVariables: [
+        [key: 'x-github-event', regexpFilter: '']
+      ],
+      causeString: "Triggered on ${repo_name}#${pr_id}: ${pr_title}",
+      //Allow to use a credential as a secret to trigger the webhook
+      //tokenCredentialId: '',
+      printContributedVariables: true,
+      printPostContent: false,
+      silentResponse: true,
+      regexpFilterText: "$pr_head_ref-$x_github_event",
+      regexpFilterExpression: '^(refs/tags/current|refs/heads/master/.+)-comment$'
+    )
+  }
+  stages {
+    stage('Process GitHub Event') {
+      when {
+        beforeAgent true
+        anyOf {
+          expression { env.action == "edited" }
+          expression { env.action == "opened" }
+          expression { env.action == "reopened" }
+          expression { env.action == "synchronize" }
```

Comment:
We need to create a new expression to capture GH comments: i.e. `/run-fleet-e2e-tests`